scripts/modules/cli.py

```diff
@@ -363,7 +363,6 @@ def init_start_parser(self, parser, services, argv=None):
             action="store",
             help="APM log level to use",
             choices=["off", "error", "warn", "info", "debug", "trace"],
-            default=DEFAULT_APM_LOG_LEVEL
```

Comment:
All uses of this setting use `self.options.get` with a default,
so we can remove the default in `argparse`. This allows the python
agent to only use this setting if it's explicitly set, which is better
for how the python agent's `log_level` works.